### DISCUSSION TITLE
Cannot build freetz-ng for 6660 with 7.90

### DISCUSSION THREAD

**Discussion by Er4twXz** [Freetz-NG/freetz-ng#1000] [Allgemein]
Message is:
`Freetz-NG 23861-5069f6ef6 master 2024-06-12
JUIS: Using cached value ... No cached value, asking avm for latest firmware ... no valid answer, trying backup ... make[1]: *** [make/include/700-image.mk:146: dl/fw/6660-de_labor.detected.image] Terminated
make: *** [Makefile:47: envira] Terminated
`

**Comment by Er4twXz** [comment]
It turns out that I had "detect latest firmware" turned on. After turning it off it started compiling.

I am running 7.90-113461 BETA after changing the version values in the firmware.ini

**Comment by fda77** [comment]
avm answers are unreliable
